Client/src/modeloClases/Entrenador.java

```diff
@@ -41,32 +58,102 @@ public int getCartasDisponibles() {
 		return cartasDisponibles;
 	}
 
+	public int getClasificacionActual() {
+		return clasificacionActual;
+	}
+
+	public int getCreditos() {
+		return creditos;
+	}
+
 	//verificar el -1 indice
+    /**
+     *M�todo encargado de elegir un pok�mon al azar de la lista.<br>
+     *<b> Pre: </b> La lista pokemones no debe estar vac�a.<br>
+     * @return Retorna un pokemon al azar. 
+     */
     public Pokemon eligePokemon(){
         int indiceRandom = new Random().nextInt(pokemones.size()-1);
         return pokemones.get(indiceRandom);
     }
     
     
+    /**
+     * M�todo que permite a un entrenador utilizar una de sus cartas de manera aleatoria, afectando un objeto IHechizable.<br>
+     * <b> Pre: </b> El par�metro pokemon debe ser distinto de null.<br>
+     * <b> Post: </b> El IHechizable par�metro ser� hechizado por una carta y el Entrenador tendr� 1 carta menos para utilizar.<br>
+     * @param pokemon, de tipo IHechizable : Representa el objeto a hechizar con una carta.
+     */
     public void usarCarta(IHechizable pokemon){
         int randomCarta = new Random().nextInt(cartas.size()-1);
         cartas.get(randomCarta).hechizar(pokemon);
         this.cartasDisponibles--;
         
     }
     
-    
-    public void obtienePremio(){
+    /**
+     * M�todo encargado de otorgarle cr�ditos al Entrenador cuando gana una batalla.
+     */
+    public void obtienePremio(){ //recordar a�adirlo a batalla
         this.creditos +=1000;
     }
 
+    /**
+     *M�todo encargado de actualizar e incrementar la clasificaci�nActual de un entrenador.<br>
+     *<b> Pre: </b> La lista de pokemones no debe estar vac�a.<br>
+     *<b> Post: </b> Puede que el entrenador aumente su clasificacion, mejorando los atributos de sus pokemones.<br>
+     */
     @Override
 	public void actualizaClasificacion() {
+		int auxClasificacion = 0;
+
+		for (Pokemon itPokemones : this.pokemones) 
+			auxClasificacion += itPokemones.clasificacionActual;
```

Comment:
```suggestion
			auxClasificacion += itPokemones.getClasificacionActual();
```

El pokemon deberia de ser el encargado de manejar y modificar sus propios atributos, no el entrenador. Rompe con el principio de Encapsulamiento